plugins.json

```diff
@@ -25,5 +25,32 @@
     },
     "version": "0.0.10",
     "url": "https://github.com/massalabs/thyra-node-manager-plugin"
+  },
+  {
+    "name": "MassaWallet",
+    "author": "MassaLabs",
+    "logo": "logo.svg",
+    "massaStationVersion": ">=0.1.14",
+    "description": "Massa Labs official wallet",
+    "assets": {
+      "windows": {
+        "url": "https://github.com/massalabs/thyra-plugin-wallet/releases/download/v0.1.1/wallet-plugin_windows-amd64.zip",
```

Comment:
by have the whole URL, we keep the fredom. We could put only the zip filename and make MassaStation concatenate but I believe we should keep it generic.

So I would say it's ok, let's keep it like this.

Reply:
>by have the whole URL, we keep the fredom.

Why we are losing freedom in having one source of truth!? 🤔 

Anyway, what I would like to avoid is having a broke link because we just forget to change one of all those link. If this day arrives you'll remember this tip. 👼

Reply:
What I meant is: imagine we want temporary to point to something else than github release asset for a specific OS, we could not with generic assets URL.

splitting the URL won't enforce the link to not break anyway

What you propose could be a good idea but because of the reason I give above and the rework it would require, I think we should keep plain URL and check for no breaking link during code review

Reply:
> What I meant is: imagine we want temporary to point to something else than github release asset for a specific OS, we could not with generic assets URL.

Got your point! Fair enough! 👌